### ISSUE TITLE
Temporarily disable all addons

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6570]
### Why this would be helpful

When trying to find the environment where a bug happens, it can be annoying to back up the addons and disable all of the ones that could affect the bug.

### My suggestion

Add a feature to the settings page that temporarily disables all addons.

### Possible alternatives

It could also just disable all of them, but doing it temporarily would be more convenient.

**Comment by WorldLanguages** [comment]
Some users don't know they can disable the whole extension from `chrome://extensions`.

**Comment by WorldLanguages** [comment]
Related: #5203

**Comment by Samq64** [comment]
> Some users don't know they can disable the whole extension from `chrome://extensions`.

There should probably be added to the FAQ.

**Comment by WorldLanguages** [comment]
> There should probably be added to the FAQ.

I think we actually should consider a "pause addons" button

**Comment by Samq64** [comment]
> I think we actually should consider a "pause addons" button

Like a disable for 15 minutes type of thing?

**Comment by mybearworld** [comment]
> > I think we actually should consider a "pause addons" button
> 
> Like a disable for 15 minutes type of thing?

It wouldn't need to be timed.

**Comment by WorldLanguages** [comment]
I think most adblockers have "pause" features, we can probably take inspiration from those.

**Comment by Samq64** [comment]
What about putting a toggle in the header bar that dims the rest of the page but still lets the user interact with it?

**Comment by apple502j** [comment]
@WorldLanguages Adblocker pauses per website, so this isn't a good example.

**Comment by MaterArc** [comment]
Of course the icon would match and fit better, but you get the idea...

**Comment by mxmou** [comment]
In #3496, I suggested a switch to the "Enabled" group.

**Comment by WorldLanguages** [comment]
> Some users don't know they can disable the whole extension from `chrome://extensions`.

+1 from feedback

**Comment by WorldLanguages** [comment]
> > Some users don't know they can disable the whole extension from `chrome://extensions`.
> 
> +1 from feedback

+1 from discussion: https://github.com/ScratchAddons/ScratchAddons/discussions/7408

**Comment by WorldLanguages** [comment]
> Some users don't know they can disable the whole extension from `chrome://extensions`.

@DNin01 Do you remember why you negatively reacted to this comment?

**Comment by DNin01** [comment]
> > Some users don't know they can disable the whole extension from `chrome://extensions`.
> 
> @DNin01 Do you remember why you negatively reacted to this comment?

I was just surprised.

I'll change my reaction to 👀.

**Comment by DNin01** [comment]
I guess anyone could not know something that seems simple, including how to access the Scratch Addons settings page. It's an understandable problem, but also easy to overlook when you've grown up with tech.

**Comment by WorldLanguages** [comment]
> I guess anyone could not know something that seems simple, including how to access the Scratch Addons settings page.

Yes, that is why I worked on some enhancements so that googling "scratch addons settings" can lead users to the settings page.

**Comment by WorldLanguages** [comment]
+1 from feedback:

> can you plssss add a way to disable all the addons with some sort of on and off button but make sure it saves so it is not annoying af

**Comment by WorldLanguages** [comment]
Search keywords: disable all addons, pause extension, adblock, adblocker, adblockers, turn off, every addon, extension page
